vk-parse/src/parse.rs

```diff
@@ -480,7 +481,8 @@ fn parse_type<R: Read>(ctx: &mut ParseCtx<R>, attributes: Vec<XmlAttribute>) ->
         "allowduplicate" => allowduplicate = Some(a.value),
         "objtypeenum"    => objtypeenum    = Some(a.value),
         "bitvalues"      => bitvalues      = Some(a.value),
-        "comment"        => comment        = Some(a.value)
+        "comment"        => comment        = Some(a.value),
+        "deprecated"     => deprecated     = a.value.parse().ok(),
```

Comment:
TODO: Need to append this exception to `$ctx` before setting `deprecated` to `None`; but it doesn't seem `FromStr` with `.parse()` has been used before.